isis/tests/CamptFunctionalTests.cpp

```diff
@@ -137,3 +140,125 @@ TEST_F(DefaultCube, FunctionalTestCamptDefaultParameters) {
   EXPECT_NEAR( toDouble(groundPoint.findKeyword("LookDirectionCamera")[1]),  2.5816511718707e-05, 1e-8);
   EXPECT_NEAR( toDouble(groundPoint.findKeyword("LookDirectionCamera")[2]),  0.99999999154535, 1e-8);
 }
+
+TEST_F(DefaultCube, FunctionalTestCamptSetSL) {
+  QVector<QString> args = {"sample=25.0", "line=25.0"};
+  UserInterface options(APP_XML, args);
+  Pvl appLog;
+
+  campt(testCube, options, &appLog);
+  PvlGroup groundPoint = appLog.findGroup("GroundPoint");
+
+  EXPECT_DOUBLE_EQ( (double) groundPoint.findKeyword("Sample"), 25.0);
+  EXPECT_DOUBLE_EQ( (double) groundPoint.findKeyword("Line"), 25.0);
```

Comment:
Since it was an issue in the other campt tests, are you confident that these wouldn't fail on other OSes due to floating-point inaccuracies?

Reply:
Since we are setting the sample and line, is it safe to assume that there would be differences? Everything is passing so far.